pkg/api/types.go

```diff
@@ -754,6 +754,8 @@ type EndpointsList struct {
 type NodeSpec struct {
 	// Capacity represents the available resources of a node
 	Capacity ResourceList `json:"capacity,omitempty"`
+	// Cidr represents the IP range assigned to the node
+	Cidr string `json:"cidr,omitempty"`
```

Comment:
Please add a note (comment) that this may need to be revisited when we support migratable IPs. I currently think that even with migratable IPs we'll want to shard the initial assignment space by host and then just not immediately return migrated IPs to the host's allocation pool, but we'll see when we actually need to implement it.